src/fundamental/macro-fundamental.h

```diff
@@ -345,6 +345,21 @@ static inline size_t ALIGN_TO(size_t l, size_t ali) {
                 ((l) + (ali) - 1) & ~((ali) - 1),                      \
                 VOID_0)
 
+/* Similar to ((t *) (void *) (p)) to cast a pointer. The macro
+ * asserts that the pointer has a suitable alignment for type "t".
+ * This exists for places where otherwise "-Wcast-align=strict"
+ * would issue a warning and if you want to assert that the cast
+ * gives a pointer of suitable alignment. */
+#define PTRFORCEALIGN(t, p)                                             \
+        ({                                                              \
+                const void * const _p = (p);                            \
+                                                                        \
+                assert((((uintptr_t) _p) % __alignof__(t)) == 0);       \
```

Comment:
redundant `()`? I think the following is enough.
`assert(((uintptr_t) _p) % __alignof__(t) == 0);`

Reply:
Yes, I like explicit parentheses around arithmetic and cast operations.

Anyway. Dropped.